scripts/azure-pipelines.yml

```diff
@@ -11,15 +11,20 @@ variables:
   FEATURE_NAME: ''
   PREVIEW_LABEL: 'rc'
   NATIVE_LINUX_PACKAGES: curl mono-complete msbuild python git libfontconfig1-dev clang-3.8 make
-  TIZEN_LINUX_PACKAGES: libxcb-xfixes0 libxcb-render-util0 libwebkitgtk-1.0-0 libxcb-image0 acl libsdl1.2debian libv4l-0 libxcb-randr0 libxcb-shape0 libxcb-icccm4 libsm6 gettext rpm2cpio cpio bridge-utils openvpn
-  MANAGED_LINUX_PACKAGES: ttf-ancient-fonts
-  MONO_VERSION: 5_18_1
-  XCODE_VERSION: 10.2.1
+  TIZEN_LINUX_PACKAGES: mono-complete msbuild libxcb-xfixes0 libxcb-render-util0 libwebkitgtk-1.0-0 libxcb-image0 acl libsdl1.2debian libv4l-0 libxcb-randr0 libxcb-shape0 libxcb-icccm4 libsm6 gettext rpm2cpio cpio bridge-utils openvpn
+  MANAGED_LINUX_PACKAGES: mono-complete msbuild ttf-ancient-fonts
+  MONO_VERSION_MACOS: 5_18_3
+  MONO_VERSION_LINUX: stable-xenial/snapshots/5.20.1.34
+  XCODE_VERSION: 10.3
+  DOTNET_VERSION: 2.2.108
   # CONFIGURATION: 'Release'
   VM_IMAGE_WINDOWS: vs2017-win2016
   VM_IMAGE_MAC: macos-10.14
   VM_IMAGE_LINUX: ubuntu-16.04
 
+  # To speed up builds
+  DOWNLOAD_EXTERNALS: '6228'
```

Comment:
In order to get any work done, There is now a way to tell DevOps to rather download the externals instead of building them. A build takes 1.5-2 hours; a download is 10 minutes.